x/ibc/applications/transfer/keeper/relay.go

```diff
@@ -186,7 +214,28 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 
 		// unescrow tokens
 		escrowAddress := types.GetEscrowAddress(packet.GetDestPort(), packet.GetDestChannel())
-		return k.bankKeeper.SendCoins(ctx, escrowAddress, receiver, sdk.NewCoins(token))
+		if err := k.bankKeeper.SendCoins(ctx, escrowAddress, receiver, sdk.NewCoins(token)); err != nil {
+			return err
+		}
+
+		defer func() {
+			telemetry.SetGaugeWithLabels(
+				[]string{"ibc", types.ModuleName, "packet", "receive"},
+				float32(data.Amount),
+				[]metrics.Label{telemetry.NewLabel("denom", unprefixedDenom)},
+			)
+
+			telemetry.IncrCounterWithLabels(
+				[]string{"ibc", types.ModuleName, "receive"},
+				1,
+				[]metrics.Label{
+					telemetry.NewLabel("chain_id", ctx.ChainID()),
```

Comment:
why the chain ID? this would never change

Reply:
👍 although it would be useful to retrieve the `chain_id` from the counterparty chain? Would that be better?

Reply:
yea that makes a lot more sense